include/graphics.c

```diff
@@ -117,6 +145,49 @@ void InitStdText() {
     SDL_BlitSurface(copyright, NULL, bgSurface, &copyrightPos);
     SDL_FreeSurface(copyright);
 
+    SDL_Rect controls1Pos = {windowWidth-170, 310, 100, 60};
+    SDL_Surface *controls1 = TTF_RenderText_Blended_Wrapped(ultraSmallFont, "D-Pad or arrow keys to move", textColor, 120);
+    SDL_BlitSurface(controls1, NULL, bgSurface, &controls1Pos);
+    SDL_FreeSurface(controls1);
+
+    SDL_Rect controls2Pos = {5, 320, 100, 60};
+    SDL_Surface *controls2 = TTF_RenderText_Blended_Wrapped(ultraSmallFont, "A or Enter to Confirm selection", textColor, 120);
+    SDL_BlitSurface(controls2, NULL, bgSurface, &controls2Pos);
+    SDL_FreeSurface(controls2);
+
+}
+
+int calculatePos(int num, int width1, int width2) {
+    // If the number is out of bounds, return maximum/minimum possible
+    if (num > 2) {
+        return (2*width1)+(2*width2);
+    }
+    if (num < 0) {
+        return (0*width1)+(0*width2);
+    }
+
+    return (num*width1)+(num*width2);
+}
+
+void DrawLetter(int x, int y, char letter) {
```

Comment:
Parameter(and the calculation function) are hard to understand without a context, I'm suggesting adding some comments about whats happening.